site/data/dynamo.ts

```diff
@@ -485,9 +485,10 @@ export const insertPublishedDisruptionIntoDynamoAndUpdateDraft = async (
                             PK: id,
                             SK: `${disruption.disruptionId}#INFO`,
                         },
-                        UpdateExpression: "SET publishStatus = :1",
+                        UpdateExpression: "SET publishStatus = :1, lastUpdated = :2",
                         ExpressionAttributeValues: {
                             ":1": status,
+                            ":2": getDate().toISOString(),
```

Comment:
I think this needs adding to the `upsertDisruptionInfo` function too. When I tried creating a disruption it didn't add the "lastUpdated" field"